src/runtime_src/core/pcie/tools/xbmgmt/xmc.cpp

```diff
@@ -500,15 +507,52 @@ int XMC_Flasher::writeReg(unsigned RegOffset, unsigned value) {
     return 0;
 }
 
+static std::string getStatus(int status, std::map<int, std::string> &map)
```

Comment:
Move this after the maps. No need to pass in the map.

Reply:
fixed